src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -105,7 +105,7 @@ picked time zone, but you will have to convert it yourself when necessary.
 			v-model="time"
 			type="datetime"
 			:show-timezone-select="true"
-			:timezone-id.sync="tz" /><br>
+			v-model:timezone-id="tz" /><br>
```

Comment:
v-model is used twice?

Reply:
Yes, that's a change of vue 3, see here: https://v3-migration.vuejs.org/breaking-changes/v-model.html
`NEW: Multiple v-model bindings on the same component are possible now;`

Reply:
With different arguments. First has default 'modelValue` argument and the second - `timezoneId`, so they set and listen for different props/events.